src/lib/Form/Data/Content/Translation/MainTranslationUpdateData.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Data\Content\Translation;
+
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @todo Add validation.
+ */
+class MainTranslationUpdateData
+{
+    /**
+     * @Assert\NotBlank()
+     *
+     * @var ContentInfo|null
+     */
+    public $contentInfo;
+
+    /**
+     * @Assert\NotBlank()
+     *
+     * @var string
+     */
+    public $languageCode;
+
+    /**
+     * @param ContentInfo|null $contentInfo
+     * @param string|null $languageCode
+     */
+    public function __construct(
+        ContentInfo $contentInfo = null,
```

Comment:
Since `$contentInfo` property is nullable it should allowing passing nulls directly too: `?ContentInfo $contentInfo = null`